src/Flexible.php

```diff
@@ -168,6 +168,12 @@ public function preset($classname)
         return $this;
     }
 
+    public function collapsed(bool $value)
```

Comment:
Same here actually, using a default value could be nice.

If you do
```php
public function collapsed(bool $value = true)
```

Then when you define the field you can do 

```php
Flexible::make('My flexible')->collapsed()

// which reads nicer than

Flexible::make('My flexible')->collapsed(true)

// But you can still do

Flexible::make('My flexible')->collapsed(false)
```

Now if everything is collapsed by default, I don't know if it would be very useful at all, but it does read nicer.

Reply:
I've added this.

I have it be default to be collapsed, I can update it to not be if you like. I'm using it heavily and collapsing it makes more sense but that might not be for everyone.